chat.go

```diff
@@ -13,6 +13,25 @@ import (
 // about a chat command that's taking long to execute.
 var ChatCmdTimeout = 10 * time.Second
 
+// Exec handles a string as if it where send by a client.
+// string is imediate returnvalue of chatcommand (if applicable)
+// bool   is if msg is getting forwarded to the minetest server
+func (cc *ClientConn) Exec(msg string) (string, bool) {
+	pkt := &mt.ToSrvChatMsg{
+		Msg: msg,
+	}
+
+	str, fwd := onChatMsg(cc, pkt)
```

Comment:
This can block, make it concurrent just like regular chat handling

Reply:
can't cuz return values

Reply:
Can but I guess it’s not that important for now. Resolving.